rep-0145.rst

```diff
@@ -0,0 +1,207 @@
+
+  - The world frame orientation depends on the IMU device, and is not explicitly defined in the transform tree.
+
+* The device has an associated *neutral orientation*, defined as the orientation of the device where the body and the world frame align [3]_.
+
+Data Reporting
+--------------
+
+* To maintain interoperability with ROS conventions, all frames must be right handed. If any data is reported left handed the driver must convert it to right handed by inverting the `y` axis.
+
+* All data from a sensor should be published with respect to a single consistent body frame. If any data is reported in an inconsistent frame of reference relative to the other data, the driver must transform it into the body frame before publishing.
+
+* Otherwise, all data should be published by the driver as it is reported by the device. Any subsequent modifications to the data (e.g. filtering, transformations) should be delegated to a downstream consumer of the data [3]_.
+
+* A prominent note should be made in the driver documentation regarding any internal data manipulation that does not comply with the requirements in this document.
+
+Raw Data
+''''''''
+
+* Accelerometers
+
+  - The accelerometers report linear acceleration data in the body frame of the device. This data is output from the driver as a 3D vector, with the components representing the deflection of the internal accelerometers.
+
+  - When the device is at rest, the vector will represent the deflection solely due to gravity, and will always point 'up' away from the earth's gravitational center. This data must be in m/s^2.
+
+* Gyroscopes
+
+  - The gyroscopes report rotational velocity data in the body frame of the device. This data is output from the driver as a 3D vector, with the components representing the instantaneous velocity around each equivalent axis of the body frame.
+
+  - The rotational velocity is right handed with respect to the bode axes, and independent of the orientation of the device. This data must be in rad/s.
+
+* Magnetometers
+
+  - The magnetometers report magnetic field strength in the body frame of the device. This data is output from the driver as a 3D vector, with the components representing magnetic field strength in each direction. This data must be in Tesla.
+
+
+Filtered Data
+'''''''''''''
+
+* Orientation
+
+  - The IMU sensor may provide a fused orientation estimate. This data is output from the driver in the form of a quaternion, which represents the orientation of the body frame in the world frame.
+
+  - In the neutral orientation, the body frame is aligned with the world frame, so the orientation will be the identity quaternion.
+
+
+Transformation
+--------------
+
+Applying a transformation to IMU data requires applying an identical rotation to both the body and the world frames. After a transformation from one body frame to another, the output data represents the output of a simulated IMU having the new body and world frames.
```

Comment:
Thanks, will test and add to https://github.com/ros/geometry_experimental/pull/78. Considering taking the whole transformation section out, as it seems out of scope for the REP.